src/site/content/en/lighthouse-performance/total-blocking-time/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Total Blocking Time
+description: |
+  Learn about Lighthouse's Total Blocking Time metric and
+  how to measure and optimize it.
+web_lighthouse:
+  - total-blocking-time
+date: 2019-10-09
+---
+
+Total Blocking Time (TBT) is one of the metrics tracked in the **Performance** section
+of the Lighthouse report. Each metric captures some aspect of page load speed.
+
+The Lighthouse report displays TBT in milliseconds:
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="total-blocking-time.jpg" alt="A screenshot of the Lighthouse Total Blocking Time audit">
+</figure>
+
+## What TBT measures
+
+TBT measures the total amount of time that a page is blocked from responding to user input,
+such as mouse clicks, screen taps, or keyboard presses. The sum is calculated by adding
+the duration of all [long tasks] between [First Contentful Paint][fcp] and [Time to Interactive][tti].
+Any task that executes for more than 50 milliseconds is a long task.
+
+## How Lighthouse determines your TBT score
+
+Your TBT score is a comparison of your page's TBT time and TBT times for the top 10,000 sites
+when loaded on mobile devices. The top site data includes 404 pages.
+
+This table shows how to interpret your TBT score:
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>TBT time<br>(in milliseconds)</th>
+        <th>Color-coding</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>0–200</td>
```

Comment:
green starts at around ~300, the PODR in the source is roughly a score of 98 :)

Reply:
Done